manifest.yaml

```diff
@@ -1,7 +1,7 @@
 id: lnbits 
 title: "LNBits"
-version: 0.9.4
-release-notes: "Update to v0.9.4 and to run on x86_64 architecture"
+version: 0.9.5
```

Comment:
should be 0.9.5.1 https://github.com/lnbits/lnbits/tree/0.9.5.1

Reply:
fixed